code/game/objects/structures/electricchair.dm

```diff
@@ -63,13 +57,16 @@
 	s.set_up(12, 1, src)
 	s.start()
 	if(buckled_mob)
-		buckled_mob.burn_skin(85)
+		buckled_mob.burn_skin(110)
 		to_chat(buckled_mob, SPAN_DANGER("You feel a deep shock course through your body!"))
 		sleep(1)
-		buckled_mob.burn_skin(85)
+		buckled_mob.burn_skin(110)
 		buckled_mob.Stun(600)
-	visible_message(SPAN_DANGER("The electric chair went off!"), SPAN_DANGER("You hear a deep sharp shock!"))
+	visible_message(SPAN_DANGER("The electric chair went off!"))
+	visible_message(SPAN_DANGER("You hear a deep sharp shock!"))
```

Comment:
This second param for visible message is what you hear when you're blind. This is just playing both messages for everyone.

Reply:
Thanks for your review. My bad, I did not know that. I reverted this modification.